app.js

```diff
@@ -23,6 +28,32 @@ app.use(function (req, res, next) {
     // Pass to next layer of middleware
     next();
 });
+//TODO create the client using the actual host when connected to dev or prod, do this by adding the host and port into the createclient() function
+var client = redis.createClient();//CREATE REDIS CLIENT
+
+
+
+//get and parse cookie and place it into req.cookies
+app.use(cookieParser());
+app.use(function(req,res,next)
+{
+
+    console.log(req.cookies.auth);
+    client.exists(req.cookies.auth, function(err, reply) {
```

Comment:
We should name the cookie "IMPId", many cookies can be set in a users browser at once from many websites and we want to make sure ours does not collide with other cookie names.